t/ppi_element_replace.t

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/perl
+
+# Unit testing for PPI::Element
+
+use lib 't/lib';
+use PPI::Test::pragmas;
+use Test::More tests => 14;
```

Comment:
I've bumped the version of `Test::More`, so it should be fine to call `done_testing()` rather than having an explicit `plan`.